packages/trace-viewer/src/ui/callTab.tsx

```diff
@@ -27,50 +27,68 @@ import type { ActionTraceEventInContext } from './modelUtil';
 
 export const CallTab: React.FunctionComponent<{
   action: ActionTraceEventInContext | undefined,
+  executionStartTime: number,
+  executionStartWallTime: number,
   sdkLanguage: Language | undefined,
-}> = ({ action, sdkLanguage }) => {
+}> = ({ action, executionStartTime, executionStartWallTime, sdkLanguage }) => {
+  // We never need the waitForEventInfo (`info`).
+  const paramKeys = React.useMemo(() => Object.keys(action?.params ?? {}).filter(name => name !== 'info'), [action]);
+
   if (!action)
     return <PlaceholderPanel text='No action selected' />;
-  const params = { ...action.params };
-  // Strip down the waitForEventInfo data, we never need it.
-  delete params.info;
-  const paramKeys = Object.keys(params);
-  const timeMillis = action.startTime + (action.context.wallTime - action.context.startTime);
-  const wallTime = new Date(timeMillis).toLocaleString();
+
+  // Calculate execution time relative to the test runner's start time
+  const startTimeMillis = action.startTime - executionStartTime;
+  const startTime = msToString(startTimeMillis);
+
+  const wallTimeMillis = startTimeMillis + executionStartWallTime;
```

Comment:
This would be just a guess at this point as we can't add ticks to millis, do we really want to explicitly mislead? I would omit.

Reply:
Offline discussion with Yury where I am saying that we should not do wall time math: maybe we could formalize model.startTime and then we can remove the wallTime from individual actions and operate in terms or ticks (monotonic time) for actions only.